# Single-Cell
## New Techniques
*  [Slide-seq](https://www.broadinstitute.org/news/new-tool-uses-rna-sequencing-chart-rich-maps-cellular-and-tissue-function?utm_source=twitter&utm_content=news-story%2Ctools&utm_medium=social&utm_campaign=Broad+Institute) A scalable technology for measuring genome-wide expression at high spatial resolution. [article](https://sci-hub.tw/10.1126/science.aaw1219)
* [course](https://hemberg-lab.github.io/scRNA.seq.course/construction-of-expression-matrix.html)


## UMI
* [UMI counting issues](https://hemberg-lab.github.io/scRNA.seq.course/construction-of-expression-matrix.html)
![image](img/UMI.issues.png)

## Dimensionality reductions
*  [PCA plot](https://hemberg-lab.github.io/scRNA.seq.course/cleaning-the-expression-matrix.html#visual-pca)ï¼Œ Convert a set of observations into a set of values of linearly uncorrelated variables called principal components 
![PCA](img/PCA.png)
*  [tSNE map](https://hemberg-lab.github.io/scRNA.seq.course/cleaning-the-expression-matrix.html#visual-tsne)
*  [MDS plot]

## Clustering of cells
*  [Clustering methods](https://hemberg-lab.github.io/scRNA.seq.course/biological-analysis.html)
*  [Clustering packages and softwares](https://hemberg-lab.github.io/scRNA.seq.course/biological-analysis.html#clustering-introduction)
*  [SC3](https://github.com/hemberg-lab/SC3)
*  [pcaReduce](https://github.com/JustinaZ/pcaReduce)

## experimental issues
*  Appropriate approaches to batch effects in scRNASeq. Red arrows indicate batch effects which are (pale) or are not (vibrant) correctable through batch-correction
![batch effects](img/batch.effects.png)

## Datasets
*  [Tabula Muris](https://tabula-muris.ds.czbiohub.org/) contains nearly 100,000 cells from 20 organs and tissues in Mus musculus. [github](https://github.com/czbiohub/tabula-muris). [dataset download](https://figshare.com/projects/Tabula_Muris_Transcriptomic_characterization_of_20_organs_and_tissues_from_Mus_musculus_at_single_cell_resolution/27733)
*  [summary of scRNA-seq tools](https://www.scrna-tools.org/)
